spring-cloud-stream-modules-docs/src/main/asciidoc/sources.adoc

```diff
@@ -102,6 +102,48 @@ $$messageSize$$:: $$the size of message to send$$ *($$Integer$$, `1000`)*
 $$producers$$:: $$the number of producers$$ *($$Integer$$, `1`)*
 $$outputType$$:: $$how this module should emit messages it produces$$ *($$MimeType$$, default: no default)*
 
+[[spring-cloud-stream-modules-source-rabbit]]
+=== RabbitMQ
+
+The "rabbit" source enables receiving messages from RabbitMQ.
+
+The queue(s) must exist before the stream is deployed; they are not created automatically.
+You can easily create a Queue using the RabbitMQ web UI.
+
+==== Options
+
+The **$$rabbit$$** $$source$$ has the following options:
+
+$$enableRetry$$:: $$enable retry; when retries are exhausted the message will be rejected; message disposition will depend on dead letter configuration$$ *($$boolean$$, default: `false`)*
+$$initialRetryInterval$$:: $$initial interval between retries$$ *($$int$$, default: `1000`)*
+$$mappedRequestHeaders$$:: $$request message header names to be propagated to/from the adpater/gateway$$ *($$String$$, default: `STANDARD_REQUEST_HEADERS`)*
+$$maxAttempts$$:: $$maximum delivery attempts$$ *($$int$$, default: `3`)*
+$$maxConcurrency$$:: $$the maximum number of consumers$$ *($$int$$, default: `1`)*
+$$maxRetryInterval$$:: $$maximum retry interval$$ *($$int$$, default: `30000`)*
+$$queues$$:: $$the queue(s) from which messages will be received$$ *($$String$$, default: `<stream name>`)*
+$$requeue$$:: $$whether rejected messages will be requeued by default$$ *($$boolean$$, default: `true`)*
```

Comment:
Yes - it's a dilemma - Seems wrong to replicate the default in the properties but I suppose it won't hurt.